Toshi/Source/Toshi/Core/TScheduler.cpp

```diff
@@ -1,7 +1,29 @@
 #include "pch.h"
 #include "TScheduler.h"
+#include "TTask.h"
 
 namespace Toshi
 {
 	TOSHI_CLASS_DERIVED_INITIALIZE(TScheduler, TObject, MKVERSION(1, 0))
+		
+	TTask* TScheduler::CreateTask(TClass* toshiClass, TTask* task)
```

Comment:
Do you actually get a TClass there not a TTask? Wow why

Reply:
TClass is used to create TObject from it, the created TObject should be inherited from TTask. that's how it works in JPOG